docs/criteria.md

```diff
@@ -57,7 +57,7 @@ In the case that a resource holds multiple kinds of content (software, ontologie
 
 **Rationale:** License without access is almost meaningless. The data provided by a resource needs to be available to research groups in a transparent and reasonable manner. Any particular content grouping at a resource should be available in its entirety with a single action once reasonable accommodations have been made with the data provider.
 
-- <span id="criteria-C.1">**C.1**</span>) Does the resource provide a “reasonable good-faith location” to access all data groupings with a single action, at an API endpoint or URL?
+- <span id="criteria-C.1">**C.1**</span>) Does the resource provide a “reasonable good-faith location” to access all data groupings with a single set of actions, at an API endpoint or URL?
```

Comment:
Provide an example as to what qualifies a "set of actions" otherwise could be anything.